src/OHIFStandaloneViewer.css

```diff
@@ -1,17 +1,33 @@
 html,
 body,
 #root {
```

Comment:
Because we target UMD, and this may be embedded outside of an iFrame, we need to be careful about setting root, body, and html styles the could impact an app other than our own. Generally, we should try to use our app container as much as possible, with styles like this potentially being set in our index.html or an external top level css file.

I hope that makes sense. This is probably fine for this PR, but worth noting as it will be an issue as we clean up our embedded use case.